File: problemas/iniciante/1113/1113.cs
using System;

class URI {
    static void Main(string[] args) {
        string entrada;

        while((entrada = Console.ReadLine()) != null){
            string[] numeros = entrada.Trim().Split(' ');
            int X = int.Parse(numeros[0]);
            int Y = int.Parse(numeros[1]);

            if(X == Y){
                break;
            }

            if(X < Y){
                Console.WriteLine("Crescente");
            }else{
                Console.WriteLine("Decrescente");
            }
        }
    }
}

File: problemas/iniciante/1113/1113.jl
entrada = split(readline())
numeros = [parse(Int32, numero) for numero in entrada]
x = numeros[1]
y = numeros[2]
while x != y
    if x < y
        println("Crescente")
    else
        println("Decrescente")
    end

    global entrada = split(readline())
    global numeros = [parse(Int32, numero) for numero in entrada]
    global x = numeros[1]
    global y = numeros[2]
end